### PR TITLE
Proposing blue color scheme

### PR DESCRIPTION
**Pull request by flossypurse**
Primary color is `#127ae5`
Per our Color Palette.
Shades and tints generated here: https://coolors.co/127ae5

### CODE REVIEW

**Comment by sw-yx** [comment]
imo this blue is a bit too bright still

File: src/css/custom.css [hunk 1]
```diff
@@ -79,7 +79,7 @@ body {
 }
 
 .footer {
-  background-color: var(--ifm-color-primary);
+  background-color: var(--ifm-color-primary-darkest);
```
Comment (sw-yx) [inline src/css/custom.css:82]: made it darker for more contrast - wdyt?

**Review by sw-yx — APPROVED** [review]

**Review by rylandg — APPROVED** [review]
7.5/10 but strong improvement over current

File: src/css/custom.css (hunk 1 above)
Comment (flossypurse) [reply src/css/custom.css:82]: Nice - I dig it!

**Pull request merged**